arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
  gains:
    joint_1:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_2:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_3:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_4:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_5:
      p: 100
      d: 1
      i: 1
      i_clamp: 1

  controller_list:
  - name: aarm_controller
    action_ns: follow_joint_trajectory
    type: FollowJointTrajectory
    default: True
    joints:
      - joint_1
      - joint_2
      - joint_3
      - joint_4
      - joint_5